tests/runtime/in_http.c

```diff
@@ -588,7 +588,7 @@ void flb_test_http_failure_400_bad_disk_write()
     test_ctx_destroy(ctx);
 }
 
-void flb_test_http_tag_key()
+void flb_test_http_tag_key(char *input)
```

Comment:
Could you change the function name from the current one?
This is because just using the current one is easily to get confused for the test functions names.
So, we should use the different name such as `test_http_tag_key` which is simply removing `flb_` prefix from that function.
In this test file, `flb_` prefix implies this test function should start here and should be defined it in TEST_LIST.

Reply:
Thanks, @cosmo0920. I have corrected the function name as you suggested.